pkg/brokerapi/brokerapi.go

```diff
@@ -106,9 +106,12 @@ func (b BrokerAPI) Unbind(ctx context.Context, instanceID, bindingID string, det
 
 // GetBinding fetches an existing service binding
 //   GET /v2/service_instances/{instance_id}/service_bindings/{binding_id}
+// TODO(mw): adjust to use details.PlanID when https://github.com/pivotal-cf/brokerapi/pull/138 is merged.
```

Comment:
IMO it might be even worth it to use an unreleased version of the library to make use of it. It would solve our "FindInstanceWithoutPlan" issue, right?

Reply:
"it depends" ;) 

It's not required (actually in quite a lot of those APIs it's not required). I'm not sure what the client in our production deployment does, whether it always sends the PlanID/ServiceID. 

We could use the fork but we'd still have to check first what the client does..